art/art_param.py

```diff
@@ -1423,7 +1432,16 @@
             "ninja": [ninja_dic, False],
             "tai_viet": [tai_viet_dic, False],
             "subscript2": [subscript2_dic, False],
-            "fancy130": [fancy130_dic, False]
+            "fancy130": [fancy130_dic, False],
+            "smooth3": [smooth3_dic, False],
+            "flaky": [flaky_dic, False],
+            "tiny_caps": [tiny_caps_dic, False],
+            "high_above": [high_above_dic, False],
+            "handwriting": [handwriting_dic, False],
+            "fantasy1": [fantasy1_dic, False],
+            "manga": [manga_dic, False],
+            "upside_down1": [upside_down1_dic, False],
+            "cool1": [cool1_dic, False]
```

Comment:
Why `cool1`, `upside_down1`, `fantasy1`? where is the second version!?

Reply:
About `cool` you're right I think it's because of the `font_wizard` it gives me error when I wanted to use `cool`.
But for others we have `upside_down` and `fantasy` fonts too.

Reply:
Okay, I think it would be better if rename `upside_down` and `fantasy` to`upside_down1` and `fantasy1` and new fonts to `upside_down2` and `fantasy2`

Reply:
OK